services/webhook/tests/test_walkthrough.py

```
@@ -145,12 +145,16 @@ def test_walkthrough_body_carries_marker_summary_table_diagram_effort():
         degraded=False,
     )
     assert body.startswith(MARKER)
+    assert "## Walkthrough" in body
     assert "Adds a guard clause." in body
     assert "x.py" in body and "tweak" in body
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)